about.html

```diff
@@ -15,35 +15,79 @@
   <body>
     <header class="main-header">
       <div class="container container-nav">
+        <!-- Button Humburger -->
         <button class="btn-menu">
           <ion-icon name="menu-outline" class="icon-menu"></ion-icon>
         </button>
-        <nav class="main-nav">
-          <div class="logo">
-            <a href="index.html"
-              ><img
-                class="logo"
-                src="./imgs/logo.png"
-                alt="Genius Academy Logo"
-            /></a>
-          </div>
-          <ul class="menu-bar">
-            <ion-icon name="close-outline" class="icon-menu hidden"></ion-icon>
+
+        <!-- Social Bar -->
+        <div class="nav-box">
+          <ul class="social-bar hidden">
```

Comment:
Thanks dear @AdnanOlarmmi 
- [x] I will add the Language item and for the purpose of color I used a hover effect on them, it means that when every item is hovered the color will be completely white, I thought it would be better from User Experience perspective.

Thanks 🙏